posthog/session_recordings/session_recording_v2_service.py

```diff
@@ -26,7 +30,7 @@ def listing_cache_key(recording: SessionRecording) -> str | None:
     try:
         # NB this has to be `team_id` and not `team.id` as it's called in an async context
         # and `team.id` can trigger a database query, and the Django ORM is synchronous
-        return f"@posthog/v2-blob-snapshots/recording_block_listing_{recording.team_id}_{recording.session_id}"
+        return f"@posthog/v2-blob-snapshots/v1/recording_block_listing_{recording.team_id}_{recording.session_id}"
```

Comment:
we need to bust the cache so we don't wait for 24 hours to see if this fixes anything